app/controllers/banners_controller.rb

```diff
@@ -65,7 +65,12 @@ def set_banner
   end
 
   def banner_params
-    params.require(:banner).permit(:active, :content, :name).merge(user: current_user)
+    params.require(:banner).permit(:active, :content, :name, :expires_at).merge(user: current_user)
+      .tap { |banner_params| set_expires_at_in_user_time_zone(banner_params) }
```

Comment:
This is doing two things. Can we split the call to conversion out of the `banner_params` method? You might take a look at the `CaseContactParameters` class.

Reply:
Moved this out to a `BannerParameters` class